custom/ilsgateway/migrations/0008_auto_20160401_0807.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import datetime
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ilsgateway', '0007_auto_20160322_1434'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='historicallocationgroup',
+            unique_together=None,
+        ),
+        migrations.RemoveField(
+            model_name='historicallocationgroup',
+            name='location_id',
+        ),
```

Comment:
no need to change, but am just curious why you had to drop this field first before dropping the whole model